### ISSUE TITLE
Proxy does not work when the project is built

### ISSUE THREAD

**Issue by ugur-muslim-goods** [SaeedYasin/shopify-app-template-node#10]
Hello

Thank you very much. Everyting works fine when I run in my local but when I built the project for production, frontend does not send any data to backend  and I am getting the error "There's no page at this address Check the URL and try again, or use the search bar to find what you need."

So my proxy options are like that 

 server: {
      host: host,
      port: 5007,
      hmr: hmrConfig,
      proxy: {
        "^/(\\?.*)?$": proxyOptions,
        "^/api(/|(\\?.*)?$)": proxyOptions,
        "^/callback(/|(\\?.*)?$)": proxyOptions,
        "^/assets(/|(\\?.*)?$)": proxyOptions,
      },
      
      And when I look at the logs all I see is 
      

      GET /assets/index-4825b73c.js HTTP/1.1" 200 - "https://xxxx.ngrok-free.app/?hmac=5e45ac63b1f62493392e2035374b8a06affdcf0d563875b96928708ae8bbc944&host=YWRtaW4uc2hvcGlmeS5jb20vc3RvcmUvdGVzdHVndXIxMA&shop=testugur10.myshopify.com&timestamp=1690487966" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
      

but no backend log. But if I run "pnpm run dev" in my frontend and serve from there when "Select Store button is iselected I am immidiately redirected to necessary backend endpoints like ensureInstallment and process goes flawlesly. 

So what would be the problem with npm run bild that it wouldn't redirect to backend

**Comment by SaeedYasin** [comment]
Hey, did you manage to find a solution for this? You might want to share it for everyone else as well. Thank you.

**Comment by ugur-muslim-goods** [comment]
Yes. I have resolved the issue. But it was my misunderstanding of the project what caused the probem.

I have seperate backend project and frontend project in different domains in firebase. So

I should have sent the initial request not to the frontend, because actually frontend is being pupulated from backend. 

- So I changed the domian to backend from my app settings in admin dashboard.
- I changed the static path from localhost/port to by frontend domain


 and the problem was fixed.
 
 I am closing the issue right now.

 Thanks.
